app/_src/gateway/breaking-changes/38x.md

```diff
@@ -15,4 +15,53 @@ Review the [changelog](/gateway/changelog/#3800) for all the changes in this rel
 
 ## Breaking changes and deprecations
 
-TBA
\ No newline at end of file
+### New DNS client
+Starting 3.8.x, a new DNS client library was implemented and added to {{site.base_gateway}}. The new DNS client introduces and new standardized way to configure a service, and helps improve performance. It is disabled by default, but will be enabled by default in a future release. For more information, see [Migrate to the new DNS client](/gateway/migrate-to-new-dns-client/).
```

Comment:
This isn't technically a breaking change yet, but it will be if they turn it on. Not sure that it belongs here.